### ISSUE TITLE
Broken version strings in MANIFEST.MF

### ISSUE THREAD

**Issue by bmuskalla** [remkop/picocli#1393]
While checking out some metadata in the `MANIFEST.MF`, I stumbled upon several version that look like gradle properties being stringified instead of their real value.

```
Manifest-Version: 1.0
Bnd-LastModified: 1609642549433
Bundle-ManifestVersion: 2
Bundle-Name: picocli
Bundle-SymbolicName: picocli
Bundle-Version: 4.6.1
...
Implementation-Version: task ':jar' property 'archiveVersion'
Specification-Version: task ':jar' property 'archiveVersion'
```

**Comment by bmuskalla** [comment]
Seems this got fixed in 5d0848f372aac8d6f2dd531e00597fea503f5a4f already.

**Comment by remkop** [comment]
Apologies for my late reply.
Thank you for reporting this and for finding out this has already been fixed.  :sweat_smile: 
I plan to do a release as soon as i get around to work on #1322.
